### PR TITLE
added build and test results to .gitignore

### PR DESCRIPTION
**Pull request by krichter722**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -150,3 +150,7 @@ patches
 *.orig
 *.rej
 tags
+
+confDbvHhe/
```
Comment (stgraber) [inline .gitignore]: That looks like a random name, so ignoring it probably won't do much good.

**Comment by krichter722** [comment]
Maybe we need to figure out which test or build tool is creating these file and change it to produce files/directories with a certain prefix `p` so that `p*` can be added to `.gitignore`. It's a bit annoying to have untracked file created by the build and test process.

**Comment by stgraber** [comment]
Hmm, so here running:

    ./autogen.sh
    ./configure --enable-tests
    make

With the resulting build config (all options enabled):

    ----------------------------
    Environment:
     - compiler: gcc
     - distribution: ubuntu
     - init script type(s): upstart,systemd
     - rpath: no
     - GnuTLS: yes
     - Bash integration: yes
    
    Security features:
     - Apparmor: yes
     - Linux capabilities: yes
     - seccomp: yes
     - SELinux: yes
     - cgmanager: yes
    
    Bindings:
     - lua: yes
     - python3: yes
    
    Documentation:
     - examples: yes
     - API documentation: yes
     - user documentation: yes
    
    Debugging:
     - tests: yes
     - mutex debugging: no
    
    Paths:
     - Logs in configpath: no

I end up with a clean environment:

    stgraber@dakara:~/data/code/lxc/lxc.1$ git status
    On branch lxc/master
    Your branch is up-to-date with 'github-lxc/master'.
    
    nothing to commit, working directory clean


That was with:
 - autoconf 2.69-7
 - automake 1.14.1
 - doxygen 1.8.6

Maybe different versions cause those extra files to be generated.

**Comment by krichter722** [comment]
With 6ad27c4282930c6fd003723d6e8a85d09dc44756 I have `doc/api/doxygen_sqlite3.db` after `./autogen.sh && ./configure && make`. The rest isn't reproducible.

That was with:

  * autoconf 2.69
  * automake 1.14.1
  * doxygen 1.8.9.1

**Comment by stgraber** [comment]
Confrimed that I'm also seeing the .db with recent doxygen.

**Pull request merged**